server/core/user.go

```diff
@@ -6,6 +6,7 @@ type (
 	// User represents user of the system.
 	User struct {
 		ID       uint    `gorm:"primary_key;auto_increment;not null" json:"id"`
+		Login    string  `gorm:"not null;size:255;unique_index" json:"login"`
```

Comment:
can you please remove `not null` because otherwise it won't be backward compatible, I tried running that locally and I am getting:
```
(/Users/jan/Dev/bleenco/abstruse/server/store/store.go:55)
[2021-06-17 22:24:28]  Cannot add a NOT NULL column with default value NULL
```
in the console.